backend/src/packages/chats/chat.controller.ts

```diff
@@ -96,9 +102,15 @@ class ChatController extends BaseController {
     this.addRoute({
       path: ChatsApiPath.ROOT,
       method: 'GET',
+      validation: {
+        query: findByQueryChatEntriesValidationSchema,
```

Comment:
That is a common schema for query validation, please accommodate it

Reply:
done

Reply:
done